image_geometry/include/image_geometry/pinhole_camera_model.h

```diff
@@ -1,10 +1,11 @@
 #ifndef IMAGE_GEOMETRY_PINHOLE_CAMERA_MODEL_H
 #define IMAGE_GEOMETRY_PINHOLE_CAMERA_MODEL_H
 
-#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/msg/camera_info.hpp>
 #include <opencv2/core/core.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
 #include <opencv2/calib3d/calib3d.hpp>
+#include <boost/shared_ptr.hpp>
```

Comment:
This is another place where this already exists, I just added the header include to make sure it was there.  I'll open a PR for this in the ROS1 version of the package, and then we can inherit it.